commands/last.js

```diff
@@ -0,0 +1,18 @@
+const nbaUtil = require('../util/nbaUtils')
+const yUtils = require('../util/yahooUtils')
+
+module.exports = {
+  name: 'last',
+  description: 'returns user roster for previous of current week',
+  async execute (message, args) {
+    let userRoster = await yUtils.createRosterDateMap(-6).then(resp => (resp))
+    //-6 to account for default Sunday start and -6 to get to last monday
```

Comment:
was this supposed to say // 0 to account for default Sunday start and -6 to get to last monday?